packages/cuba-react-core/src/app/MainStore.ts

```diff
@@ -7,17 +7,41 @@ export class MainStore {
 
   static NAME = 'mainStore';
 
+  /**
+   * Whether the `MainStore` instance is initialized.
+   */
   @observable initialized = false;
+  /**
+   * Whether the user authenticated.
+   */
   @observable authenticated = false;
+  /**
+   * Whether the user is anonymous.
+   */
   @observable usingAnonymously = false;
   @observable userName?: string;
+  /**
+   * Currently selected locale.
+   */
   @observable locale?: string;
 
+  /**
+   * Information about project entities.
+   */
   @observable metadata?: IObservableArray<MetaClassInfo>;
+  // TODO can be made private?
   metadataRequestCount = 0;
```

Comment:
I think we can resolve all TODO's in this file to make properties and methods private. Didn't see any cause not to do this. @web-devel ?

Reply:
Agree